docs/cody/clients/install-jetbrains.mdx

```diff
@@ -151,7 +151,7 @@ Like the Edit Code and Generate Unit Test commands, you can generate inline docu
 
 ## Supported LLM models
 
-Users on Cody **Free**, **Pro**, and **Enterprise** can choose from a list of supported LLM models for Chat and Commands. Claude Sonnet 3 is the default LLM model for Cody Free users for Chat and Commands. Users on Cody Free and Pro can choose from a list of supported LLM models for Chat and Commands.
+Users on Cody **Free** and **Pro** can choose from a list of supported LLM models for Chat and Commands. Claude Sonnet 3 is the default LLM model for Cody Free users for Chat and Commands. Users on Cody Free and Pro can choose from a list of supported LLM models for Chat and Commands.
```

Comment:
```suggestion
Users on Cody **Free** and **Pro** can choose from a list of supported LLM models for Chat and Commands. Both Cody Free and Pro users can choose from a list of supported LLM models for Chat and Commands.
```

Claude 3.5 Sonnet is the default LLM model for all users, both Free and Pro, but I think we can just remove that because both Free and Pro users can select any of the supported LLM for their tiers.